### DISCUSSION TITLE
Unable to get Github student developer pack.

### DISCUSSION THREAD

**Discussion by SevinduWarusawithana** [community/community#103147] [GitHub Education]
### Hi everyone,

I received an error when I attempted to use my university's ID card to obtain the Github student developer pack. It said that I cannot get the Github student developer pack because my university id card has the short name instead of the full name of the university. I've made multiple attempts at this, but each time my request is rejected. This Github student developer pack is very helpful for my learning. One of my friend used the old ID card back in time to purchase the Github Student Developer Pack without any issues. That friend suggested me to post a message about the inconvenience I experienced on the Github community. Because of this problem, nobody who just received a new ID card from our university has been allowed to get the Github student developer bundle. Another error was that my account was identified as belonging to a branch of our institution that was located in a different counrty when I gave my university account to Github to obtain the student delever pack. The error indicates that the country I'm in does not have the university I chose because of this problem. How can I solve this problems step by step?

**Comment by gil-son** [comment]
You must to contact [Github Support](https://support.github.com/) 

I hope I was helpfull. Good luck!

**Reply by SevinduWarusawithana** [reply]
Hi @gil-son , 
Thanks for your comment. But I have never try Github Support. Can you tell me how to contact it.

**Reply by gil-son** [reply]
Yes, sure!

Open the [Github Support](https://support.github.com/)

You will see that:

![github04](https://github.com/community/community/assets/72712095/7b1eec16-6eab-4842-915c-251bf15ff2df)

(Click on My Tickets)

After that you can create a New ticket and report it:

![github05](https://github.com/community/community/assets/72712095/c03d8c28-d26b-461d-a9c5-54937efadbd0)

I hope that can be useful!

**Reply by SevinduWarusawithana** [reply]
Oh,  I will try this. Thank you very much for helping me to resolve this problem.

**Reply by gil-son** [reply]
If it makes sense, consider marking the best answer
